src/api/built-in-directives.md

```diff
@@ -198,6 +198,10 @@ Attach an event listener to the element.
 
   When listening to native DOM events, the method receives the native event as the only argument. If using inline statement, the statement has access to the special `$event` property: `v-on:click="handle('ok', $event)"`.
 
+  When using an inline statement with the `$event` property, such as `v-on:click="handle('ok', $event)"`, the inline statement is transformed to `$event => handle('ok', $event)`. This means that the `$event` object is automatically passed as the first argument to the inline function. 
```

Comment:
We're trying to follow `show, not tell` principle in the docs, and the example below shows the behaviour you're explaining here:

```
 <button v-on:click="doThat('hello', $event)"></button>
```

I am not sure any additional clarification is needed here